code/src/terrat_vcs_service_gitlab/terrat_vcs_service_gitlab_provider.ml

```diff
@@ -2886,7 +2881,7 @@ module Comment = struct
     | `Stack_config_tag_query_err err -> raise (Failure "nyi")
 
   let publish_comment ~request_id client user pull_request =
-    let open Terrat_vcs_service_gitlab_publishers.Comment_api in
+    let open Terrat_vcs_gitlab_comment_publishers.Comment_api in
```

Comment:
Generally we don't open modules UNLESS they are providing infix operators, otherwise it becomes hard to tell which functions are coming from where.  I recommend doing `let module C = ...` or similar.

Same comment for a few diffs up.

Reply:
fixed